spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherRunner.java

```diff
@@ -45,15 +48,16 @@
 	private ModuleLauncher moduleLauncher;
 
 	@Override
-	public void run(ApplicationArguments applicationArguments) throws Exception {
-		String[] launchedModules = moduleLauncherProperties.getModules();
+	public void run(String... unused) throws Exception {
```

Comment:
That's not entirely correct, though. They're not used in the context of this method. The application arguments would still be parsed into the Environment as far as I can tell (and that's something we want). It's just that right now we process them in a different way.